mods/ra/chrome.yaml

```diff
@@ -583,6 +588,9 @@ scrollitem:
 scrollitem-hover:
 	Inherits: button
 
+scrollitem-pressed:
+	Inherits: button
```

Comment:
I'm still on the fence with regards to changing the styles for dropdowns but since #20303 is already merged you can do the same here, that way we'll get the interactive feel and it will be consistent with D2k:
```miniyaml
scrollitem-pressed:
	Inherits: button-hover
```

Reply:
It just looks bad in other mods. D2K was the only one where this solution fit

Reply:
I think it looks pretty much the same in D2k and RA/TS - clicking on a scrollitem gives you a brief flash :shrug: But I'm perfectly fine with keeping the styles from release.